example/src/CheckoutView.js

```diff
@@ -39,7 +40,7 @@ const CheckoutView = () => {
   };
 
   const didSubmit = (data, nativeComponent, configuration) => {
-    console.log('didSubmit');
```

Comment:
It is more readable if you use **template literals** instead of this approach. Example:
```javascript
console.log(`didSubmit: ${data.paymentMethod.type)}`);
```

Reply:
👍